builder/image/bin/boot

```diff
@@ -54,7 +54,7 @@ CONFD_PID=$!
 test -e /var/run/docker.sock && rm -f /var/run/docker.sock
 
 # spawn a docker daemon to run builds
-docker -d --bip=172.19.42.1/16 --insecure-registry 10.0.0.0/8 --insecure-registry 172.16.0.0/12 --insecure-registry 192.168.0.0/16 --insecure-registry 100.64.0.0/10 &
+docker -d --bip=172.19.42.1/16 --insecure-registry 10.0.0.0/8 --insecure-registry 172.16.0.0/12 --insecure-registry 192.168.0.0/16 --insecure-registry 100.64.0.0/10 --storage-driver=overlay &
```

Comment:
docker 1.5.0 on core os uses overlay fs . We have updated docker version inside builder and started docker with overlay fs
```
$ docker info
Containers: 0
Images: 0
Storage Driver: overlay
 Backing Filesystem: extfs
Execution Driver: native-0.2
Kernel Version: 3.18.6
Operating System: CoreOS 607.0.0
CPUs: 1
Total Memory: 1.959 GiB
Name: deis-03
ID: LCIO:LLZB:VOM3:T44O:HM5Y:6Z6Z:7UQT:V2UK:LNAI:HYTK:RFRZ:MOMR
```